### DISCUSSION TITLE
transparent blurry background

### DISCUSSION THREAD

**Discussion by rayan-mazouz** [hecrj/iced#1395] [Q&A]
I'm trying to make an app with a transparent background with gaussian blur, like this : 

![image](https://user-images.githubusercontent.com/91835054/182006442-ea8dff69-102b-47a8-9024-4061aa8afc2a.png)

I managed to have a transparent background with

`fn background_color(&self) -> Color {`
`····Color::TRANSPARENT`
`}`

is there any way to apply blur with iced ?

thanks

**Comment by hecrj** [comment]
Currently, there is no way to apply blur in `iced`.

**Reply by rayan-mazouz** [reply]
thanks :+1:

**Reply by unknownK19** [reply]
Blurring Windows is way complex because there are so many desktop environments that can do their own blur script. Own DE uses a different kind of shader, like `Render-to-texture (RTT)` it renders the window under dimension, converts to texture, and then does a blur effect, then uses it as a background.
_**but**_
There is a way to do some transparent with a noisy effect, but this method is very hard and not blurry.

1. Transparent the window and make sure the window is borderless.
2. Use a noisy image as a background (be sure with RGBA color) [Canvas].

> _//  Wait for new update of Wgpu which support RTT_
